lib/theme.json

```diff
@@ -186,6 +186,51 @@
 			],
 			"text": true
 		},
+		"dimensions": {
+			"defaultAspectRatios": true,
+			"aspectRatios": [
```

Comment:
These default presets are currently unused as well so the UI can continue to be grouped by portrait and landscape to match the design. It's unfortunate that [we aren't using JSON with comments](https://github.com/WordPress/gutenberg/issues/35099), so I can't really comment in the code here about it.

Reply:
I think I'm going to update the UI to use these and add a special case for the defaults to categorize them as landscape or portrait.

We could also categorize the theme supplied ones as well if we update the designs. Or we could choose to not categorize the defaults to be consistent with how other settings presets work. @richtabor maybe you have thoughts?

Reply:
> Or we could choose to not categorize the defaults to be consistent with how other settings presets work 

You mean keep the "Theme" section, with the theme's defaults, v. putting those defaults within the proper landscape/portrait sections?

Reply:
The theme ones would be as-is and the landscape and portrait get grouped into just default. For example, in the color/gradient presets, they are grouped by default and theme. And in global styles, there's an additional custom section.

![global styles color presets](https://user-images.githubusercontent.com/5129775/216399918-2cf36a43-c6a9-4629-9121-c55709076216.png)

Reply:
Is it a heavy lift to get theme presets in? If so, I'm ok dropping support for theme-specific presets (for now) and circle back around for that.

Reply:
I can organize it however you'd like. I pushed up a change to include the defaults from the default theme.json categorized under landscape and portrait already. It also isn't any more difficult to change how they're categorized if you think it makes sense to do it differently.

Reply:
Ok cool, good deal.